[README] Add all the products sponsoring open source (#14311)
mui-org/material-ui: README.md

## Thanks

> Material-UI was only made possible thanks to these great services that are sponsoring our code infrastructure:

Reviewer: ```suggestion
Material-UI is only made possible thanks to these great services that sponsor our core infrastructure:
```

- GitHub allows us to host the Git repository

[<img src="https://github.githubassets.com/images/modules/logos_page/GitHub-Logo.png" width="120">](https://github.com/)

- CircleCI allows us to run the test suite

[<img src="https://assets.brandfolder.com/otz6k5-cj8pew-e4rk9u/element.png?v=1501538594" width="120">](https://circleci.com/)

- Netfliy allows us to distribute the documentation

[<img src="https://cdn.netlify.com/15ecf59b59c9d04b88097c6b5d2c7e8a7d1302d0/1b6d6/img/press/logos/full-logo-light.svg" width="120">](https://www.netlify.com/)

> - Crowdin allows us to translation the documentation

Reviewer: ```suggestion
- Crowdin allows us to translate the documentation
```
